test/server/config_validation/server_test.cc

```diff
@@ -42,5 +81,15 @@ INSTANTIATE_TEST_CASE_P(ValidConfigs, ValidationServerTest,
                                           "google_com_proxy.yaml", "google_com_proxy.v2.yaml",
                                           "s2s-grpc-envoy.yaml", "service-envoy.yaml"));
 
+// Just make sure that all configs can be injested without a crash. Processing of config files
+// may not be successful, but there should be no crash.
```

Comment:
I'm confused. Which ones aren't successful? I think they all should be?

Reply:
I am getting bunch of errors (depending on config file):
- envoy_double_proxy.json': unable to read file: certs/clientcert.pem
- envoy_front_proxy.json': unable to open file '/var/log/envoy/outlier_events.log': No such file or directory
- envoy_service_to_service.json': unable to read file: certs/cacert.pem
- front-envoy-jaeger-v1.json': Didn't find a registered implementation for name: 'envoy.zipkin'

I think this is known problem. See: https://github.com/envoyproxy/envoy/blob/master/test/server/config_validation/server_test.cc#L36-L43

I did not try solve those problems. My goal here was not to crash when error is encountered.

What do you think if there is a directory where all sort of config files are dropped for testing? Config files reported in #2881 and #3083 could be dropped there. Also deliberately wrong configs with unknown filters, wrong json/yaml syntax. There should be no crash when processing those files.

Reply:
Ah, I see. OK. Thanks for pointing that out. OK SGTM this is better than nothing as long as we leave it well commented. Let's see if we can share setup code then LGTM.